vng_api_common/utils.py

```diff
@@ -203,3 +203,9 @@ def get_view_summary(view_cls):
         pass
 
     return ""
+
+
+def get_field_attribute(model_string: str, field_name: str, attr_name: str) -> str:
```

Comment:
where is this used?

Reply:
here https://github.com/VNG-Realisatie/gemma-zaakregistratiecomponent/pull/147/files#diff-da48c0f24e8fa0f2f2f86788e5a7d79fR57

I figured since these attributes are set on the model fields, it would be better to get them from there instead of hardcoding them